recipes/sphinxcontrib-jquery/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sphinxcontrib-jquery" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "8fb65f6dba84bf7bcd1aea1f02ab3955ac34611d838bcc95d4983b805b234daa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+    - flit-core >=3.7,<4
+  run:
+    - python >=2.7
+    - setuptools
+    - sphinx
+
+test:
+  imports:
+    - sphinxcontrib.jquery
+  commands:
+    - pip check
```

Comment:
Perhaps also run the package's tests here.

A

Reply:
@AA-Turner pinging you once more just in case the previous pings were lost in a sea of notifications.

I apologize for the large break since the last discussions, many things came up in real life here which pulled me away from GitHub.

To recap, previous discussions revealed that the package on PyPI was not packaging the tests, which made running those tests on `conda-forge`'s CI infrastructure [more complicated](https://github.com/conda-forge/staged-recipes/pull/20921#issuecomment-1294965232).

This led to [a thought](https://github.com/conda-forge/staged-recipes/pull/20921#discussion_r1008627170) that maybe it was best to leave testing to upstream (for now), and get tests running here if and when said tests were packaged into the sdist.

If that course of action sounds alright to you, we can move ahead with the simpler version of this recipe, and modify said recipe down the line. What say you?